tests-and-examples/tests/cases/metadata.rs

```diff
@@ -23,6 +25,20 @@ impl Greeter for GreeterService {
             self.tx.send((key.to_owned(), value.to_owned())).unwrap();
         }
 
+        if req.name == "root" {
+            let mut status = Status::default();
```

Comment:
Better to use `Status::with_message` in examples? I am wondering what the use case is for generating a default when these constructors are available.

Reply:
`Status` here is from protobuf, which doesn't have `with_message` constructor, instead of the one in grpcio. And using `default` is to get it compile for both prost and protobuf.